lib/DBDefs/Default.pm

```diff
@@ -103,6 +103,7 @@ sub WEB_SERVER                { "localhost:5000" }
 # Relevant only if SSL redirects are enabled
 sub WEB_SERVER_SSL            { "localhost" }
 sub LUCENE_SERVER             { "search.musicbrainz.org" }
+sub SOLR_SERVER               { "solr.musicbrainz.org/solr" }
```

Comment:
What will happen to x-accel-redirect with this change ?
See https://github.com/metabrainz/musicbrainz-server/blob/4ecff4126780c938008b5ff767d7166071d8a1f8/lib/MusicBrainz/Server/Data/WebService.pm#L211

Reply:
done